### PR TITLE
3. UI design

### PR DESCRIPTION
**Pull request by loqit**
https://loqit.github.io/ToDo/html/register.html
https://loqit.github.io/ToDo/html/tasks.html

### CODE REVIEW

File: css/nav-bar.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+}
+
+.logout-block {
+    position: absolute;
+    margin-top: 15px;
+    left: calc(100% - 110px);
+}
+
+#add-task-form {
```
Comment (gmltA) [inline css/nav-bar.css]: It's better to use classes, than IDs.

File: html/boards.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <title>ToDo</title>
+</head>
+<body>
+<header>
+    <nav></nav>
+    <div id="navbar">
+
+        <div id="app-title">
+
+            <div><h1>ToDo</h1></div>
```
Comment (gmltA) [inline html/boards.html]: Why do you need `div` here?

**Review by gmltA — CHANGES_REQUESTED** [review]

**Review by gmltA — COMMENTED** [review]
Okay. Is <kbd>8</kbd> enough?

**Comment by loqit** [comment]
> Okay. Is 8 enough?

Yes

**Review by gmltA — APPROVED** [review]

**Pull request merged**